legal_db/models.py

```diff
@@ -1,11 +1,71 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
+from django_countries.fields import CountryField
 
 
-class License(models.Model):
-    code = models.CharField(max_length=12)
-    title = models.CharField(max_length=50)
-    version = models.CharField(max_length=25, blank=True, null=True)
+class LegalResource(models.Model):
+    license = models.CharField(max_length=25, blank=True, null=True)
+    license_version = models.CharField(max_length=25, blank=True, null=True)
+    is_ported = models.BooleanField(blank=True, null=True)
+    contributor_name = models.CharField(max_length=120)
+    contributor_email = models.EmailField()
+    contribution_credit = models.BooleanField()
```

Comment:
Could we make this field name a bit clearer so that we know this refers to whether the contributor wants to be credited?

Reply:
About `contriution_credit`? I thought of `give_contribution_credit` but it seems a bit long to me, I'll go with `give_credit`, is it better this way?

Reply:
it's okay if it's long if that makes it clearer. how about `contribution_credit_requested`?

Reply:
Sound good too! :)

Reply:
Sounds good too! :)